### PR TITLE
New File Added:  MI-11-MEAN -> Sourabh_Kanwade

### PR DESCRIPTION
**Pull request by sourabh-kanwade**

### CODE REVIEW

**Review by kalyani1942 — APPROVED** [review]

**Review by kalyani1942 — APPROVED** [review]

File: Sourabh_Kanwade/html/Assignment 4/Question_2.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Table_Q2</title>
+    <style>
+      table {
+        border: 10px solid rgb(107, 105, 105);
```
Comment (Prashant698) [inline Sourabh_Kanwade/html/Assignment 4/Question_2.html]: Remove right table border. Please add CSS border-right:0px;

**Review by somesh6797 — APPROVED** [review]

**Review by dayanandviveki — APPROVED** [review]
No Changes Required

**Review by Prashant698 — APPROVED** [review]

**Review by shivling123 — APPROVED** [review]

File: Sourabh_Kanwade/html/Assignment 2/Question_1.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    <small> <i>This article is about the company.</i> </small>
+    <h2>From the garage to the Googleplex</h2>
+    The Google story begins in 1995 at <b>Stanford University</b>
+    <br />
+    <b> <i>Larry Page</i> </b>was considering Stanford for grad school and
+    <b><i>Sergey Brin,</i></b>
+    a student
+    <br />
+    there, was assigned to show him around.
+    <br />
```
Comment (shivling123) [inline Sourabh_Kanwade/html/Assignment 2/Question_1.html]: remove extra <b> opening and closing tag format code properly

check all  <b> tag in this code

Comment (sourabh-kanwade) [reply Sourabh_Kanwade/html/Assignment 2/Question_1.html]: should I replace `<br />` with `<br>`?

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2main.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    <div style="text-align: center">
+      <h1>Ambersoft</h1>
+      <br />
+      <a href="SourabhA3_2main.html">Home</a> |
+      <a href="SourabhA3_2mi11.html">MI-MEAN</a> |
+      <a href="SourabhA3_2contact.html">Contact</a>
+      <h1>Home Page</h1>
+      <br />
+    </div>
+    <br />
```
Comment (shivling123) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_2main.html]: <br> it is empty tag it doesn't have closing check it properly all br tag

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2mi11.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  <body>
+    <div style="text-align: center">
+      <h1>Ambersoft</h1>
+      <br />
+      <a href="SourabhA3_2main.html">Home</a> |
+      <a href="SourabhA3_2mi11.html">MI-MEAN</a> |
+      <a href="SourabhA3_2contact.html">Contact</a>
+      <br />
+    </div>
+    <br />
```
Comment (shivling123) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_2mi11.html]: check br tag

File: Sourabh_Kanwade/html/Assignment 2/Question_1.html (hunk 2 above)
Comment (shivling123) [reply Sourabh_Kanwade/html/Assignment 2/Question_1.html]: yes

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2mi11.html (hunk 4 above)
Comment (sourabh-kanwade) [reply Sourabh_Kanwade/html/Assignment 3/SourabhA3_2mi11.html]: done

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2main.html (hunk 3 above)
Comment (sourabh-kanwade) [reply Sourabh_Kanwade/html/Assignment 3/SourabhA3_2main.html]: done

File: Sourabh_Kanwade/html/Assignment 2/Question_1.html (hunk 2 above)
Comment (sourabh-kanwade) [reply Sourabh_Kanwade/html/Assignment 2/Question_1.html]: done

**Review by avinash0070 — COMMENTED** [review]
Chnage title to New File Added : MI-11-MEAN -> Sourabh_kanwade

File: Sourabh_Kanwade/html/Assignment 1/Question 1.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 1.html:1]: bad code indentation use 4 spaces

File: Sourabh_Kanwade/html/Assignment 1/Question 2.html [hunk 6]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 2.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 1/Question 2.html [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+<head>
+  <title>Question 2</title>
+</head>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 2.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 1/Question 2.html [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+  <p>HTML is a standard markup language for Web Pages.</p>
+  <h2>HTML Editor</h2>
+  <p>
+  <h4>Notepad or TextEdit.</h4>
+  Web pages can be created and modified by using professional HTML editors. <br>
+  However, for learning HTML we recommend a simple text editor like Notepad
+  (PC) or TextEdit (Mac).
+  </p>
+</body>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 2.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 1/Question 2.html [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+</html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 2.html]: add new line

File: Sourabh_Kanwade/html/Assignment 1/Question 2.html [hunk 10]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 2.html:1]: bad code indentation

File: Sourabh_Kanwade/html/Assignment 1/Question 3.html [hunk 11]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 3.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 1/Question 3.html [hunk 12]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<title>Question 3</title>
+</head>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 3.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 1/Question 3.html [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+	<h2>HTML Editor</h2>
+	<p>
+	<h4>Notepad or TextEdit.</h4>
+	Web pages can be created and modified by using
+	professional HTML editors. <br>
+	However, for learning HTML we recommend a simple text editor like Notepad
+	(PC) or TextEdit (Mac).
+	</p>
+</body>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 3.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 1/Question 3.html [hunk 14]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 3.html:1]: bad code indentation ask your mentor for prettier in vs code

File: Sourabh_Kanwade/html/Assignment 1/Question 3.html [hunk 15]
```diff
@@ -0,0 +1,23 @@
...
+	<p>
+	<h4>Notepad or TextEdit.</h4>
+	Web pages can be created and modified by using
+	professional HTML editors. <br>
+	However, for learning HTML we recommend a simple text editor like Notepad
+	(PC) or TextEdit (Mac).
+	</p>
+</body>
+
+</html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 3.html]: Add new line

File: Sourabh_Kanwade/html/Assignment 4/Question_3.html [hunk 16]
```diff
@@ -0,0 +1,79 @@
...
+      </tr>
+      <tr>
+        <th colspan="2">Schedule</th>
+        <th rowspan="2">Topic</th>
+      </tr>
+      <tr>
+        <th>Begin</th>
+        <th>End</th>
+      </tr>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 4/Question_3.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 4/Question_3.html [hunk 17]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 4/Question_3.html:1]: bad code indentation

File: Sourabh_Kanwade/html/Assignment 4/Question_2.html [hunk 18]
```diff
@@ -0,0 +1,116 @@
...
+    <tr>
+      <th class="border-bottom">4:30</th>
+      <td class="border-bottom"></td>
+      <td class="border-bottom"></td>
+      <td class="border-bottom"></td>
+    </tr>
+  </table>
+</body>
+
+</html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 4/Question_2.html]: add new line

File: Sourabh_Kanwade/html/Assignment 4/Question_2.html [hunk 19]
```diff
@@ -0,0 +1,116 @@
...
+    </tr>
+    <tr>
+      <th class="border-bottom">4:30</th>
+      <td class="border-bottom"></td>
+      <td class="border-bottom"></td>
+      <td class="border-bottom"></td>
+    </tr>
+  </table>
+</body>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 4/Question_2.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 4/Question_2.html [hunk 20]
```diff
@@ -0,0 +1,116 @@
...
+    }
+
+    caption {
+      font-size: large;
+      font-weight: bold;
+      color: violet;
+    }
+  </style>
+</head>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 4/Question_2.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 4/Question_2.html [hunk 21]
```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 4/Question_2.html]: Remove new line

File: Sourabh_Kanwade/html/Assignment 4/Question_2.html [hunk 22]
```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 4/Question_2.html:1]: bad code indentation

File: Sourabh_Kanwade/html/Assignment 4/Question_1.html [hunk 23]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 4/Question_1.html:1]: bad code indentation

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2mi11.html [hunk 24]
```diff
@@ -0,0 +1,48 @@
...
+    <li>Aniket</li>
+  </ul>
+  <ol>
+    <li>Bipin</li>
+    <li>Priyanka</li>
+    <li>Yashasvi</li>
+  </ol>
+</body>
+
+</html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_2mi11.html]: add new line

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2mi11.html [hunk 25]
```diff
@@ -0,0 +1,48 @@
...
+    <li>Hruday</li>
+    <li>Aniket</li>
+  </ul>
+  <ol>
+    <li>Bipin</li>
+    <li>Priyanka</li>
+    <li>Yashasvi</li>
+  </ol>
+</body>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_2mi11.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2mi11.html [hunk 26]
```diff
@@ -0,0 +1,48 @@
...
+    a:link {
+      color: #f2d4c2;
+    }
+
+    a:visited {
+      color: #d9b8cd;
+    }
+  </style>
+</head>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_2mi11.html]: Remove extra line

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2main.html [hunk 27]
```diff
@@ -0,0 +1,88 @@
...
+    a:link {
+      color: #f2d4c2;
+    }
+
+    a:visited {
+      color: #d9b8cd;
+    }
+  </style>
+</head>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_2main.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2main.html [hunk 28]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_2main.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2main.html [hunk 29]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_2main.html:1]: bad code indentation

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2contact.html [hunk 30]
```diff
@@ -0,0 +1,76 @@
...
+    us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.
+  </div>
+</body>
+
+</html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_2contact.html]: add new line

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2contact.html [hunk 31]
```diff
@@ -0,0 +1,76 @@
...
+    us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact
+    us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.
+  </div>
+</body>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_2contact.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2contact.html [hunk 32]
```diff
@@ -0,0 +1,76 @@
...
+    a:link {
+      color: #f2d4c2;
+    }
+
+    a:visited {
+      color: #d9b8cd;
+    }
+  </style>
+</head>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_2contact.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2contact.html [hunk 33]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_2contact.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_2contact.html [hunk 34]
```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_2contact.html:1]: bad code indentation

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html [hunk 35]
```diff
@@ -0,0 +1,72 @@
...
+      <li>SYSTEM-2</li>
+      <ul class="disc-list">
+        <li>Triveni</li>
+        <li>Kalyani</li>
+      </ul>
+    </ol>
+  </ol>
+</body>
+
+</html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html]: Add extra line

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html [hunk 36]
```diff
@@ -0,0 +1,72 @@
...
+      </ul>
+      <li>SYSTEM-2</li>
+      <ul class="disc-list">
+        <li>Triveni</li>
+        <li>Kalyani</li>
+      </ul>
+    </ol>
+  </ol>
+</body>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html [hunk 37]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html [hunk 38]
```diff
@@ -0,0 +1,72 @@
...
+    .mean {
+      color: peru;
+    }
+
+    .sys {
+      color: royalblue;
+    }
+  </style>
+</head>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 2/Question_1.html [hunk 39]
```diff
@@ -0,0 +1,51 @@
...
+    At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work
+    together at Google for <del>30</del> <ins>20</ins> years, until the year 2024.
+  </small>
+  <br>
+  <center style="color: #f2a365">
+    <h3>Thank You!!</h3>
+  </center>
+</body>
+
+</html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 2/Question_1.html]: add new line

File: Sourabh_Kanwade/html/Assignment 2/Question_1.html [hunk 40]
```diff
@@ -0,0 +1,51 @@
...
+    <br>
+    At that time Larry Page, Sergey Brin, and Eric Schmidt agreed to work
+    together at Google for <del>30</del> <ins>20</ins> years, until the year 2024.
+  </small>
+  <br>
+  <center style="color: #f2a365">
+    <h3>Thank You!!</h3>
+  </center>
+</body>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 2/Question_1.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 2/Question_1.html [hunk 41]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 2/Question_1.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 1/Question 4.html [hunk 42]
```diff
@@ -0,0 +1,18 @@
...
+    <h1>About Me</h1>
+    <hr />
+    <p>
+        Hello , I am <i>Sourabh</i>. I am currently persuing Computer Science
+        degree in Walchand College of Engineering, Sangli .<br>I am interested
+        in the web development and I am learning <u>HTML</u> and CSS.
+    </p>
+</body>
+
+</html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 4.html]: add new line

File: Sourabh_Kanwade/html/Assignment 1/Question 4.html [hunk 43]
```diff
@@ -0,0 +1,18 @@
...
+<body>
+    <h1>About Me</h1>
+    <hr />
+    <p>
+        Hello , I am <i>Sourabh</i>. I am currently persuing Computer Science
+        degree in Walchand College of Engineering, Sangli .<br>I am interested
+        in the web development and I am learning <u>HTML</u> and CSS.
+    </p>
+</body>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 4.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 1/Question 4.html [hunk 44]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>About Me</title>
+</head>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 4.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 1/Question 4.html [hunk 45]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 4.html]: remove extra line

File: Sourabh_Kanwade/html/Assignment 1/Question 4.html [hunk 46]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 4.html:1]: bad code indentation

File: Sourabh_Kanwade/html/Assignment 2/Question_1.html [hunk 47]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 2/Question_1.html:1]: bad code indentation

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html [hunk 48]
```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html:1]: bad code indentation

File: Sourabh_Kanwade/html/Assignment 2/Question_1.html [hunk 49]
```diff
@@ -0,0 +1,51 @@
...
+<html>
+
+<head>
+  <title>Html Formating</title>
+</head>
+
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 2/Question_1.html]: remove extra line

**Review by avinash0070 — CHANGES_REQUESTED** [review]

File: Sourabh_Kanwade/html/Assignment 2/Question_1.html (hunk 49 above)
Comment (sourabh-kanwade) [reply Sourabh_Kanwade/html/Assignment 2/Question_1.html]: done

**Comment by avinash0070** [comment]
Resolve all the comments

File: Sourabh_Kanwade/html/Assignment 1/Question 2.html (hunk 10 above)
Comment (avinash0070) [reply Sourabh_Kanwade/html/Assignment 1/Question 2.html:1]: use 4 spaces

**Review by avinash0070 — CHANGES_REQUESTED** [review]

**Comment by sourabh-kanwade** [comment]
done changes ..

File: Sourabh_Kanwade/html/Assignment 1/Question 2.html [hunk 50]
```diff
@@ -0,0 +1,18 @@
...
```
Comment (avinash0070) [inline Sourabh_Kanwade/html/Assignment 1/Question 2.html:1]: need proper code indentation

**Review by avinash0070 — CHANGES_REQUESTED** [review]

File: Sourabh_Kanwade/html/Assignment 1/Question 2.html (hunk 50 above)
Comment (sourabh-kanwade) [reply Sourabh_Kanwade/html/Assignment 1/Question 2.html:1]: done

**Review by avinash0070 — APPROVED** [review]

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html [hunk 51]
```diff
@@ -1,67 +1,72 @@
...
+      </ul>
+      <li>SYSTEM-2</li>
+      <ul class="disc-list">
+        <li>Triveni</li>
+        <li>Kalyani</li>
+      </ul>
     </ol>
-  </body>
-</html>
+  </ol>
```
Comment (shivling123) [inline Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html]: remove this ol tage

**Review by shivling123 — COMMENTED** [review]
you have changes your code ass per comment

File: Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html (hunk 51 above)
Comment (Phantom-Code) [reply Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html]: sir please check my latest commit
I am not seeing this ol in my local file

Comment (Phantom-Code) [reply Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html]: also checked on main branch . there is no `<ol>` on line 69 after `</html>

Comment (sourabh-kanwade) [reply Sourabh_Kanwade/html/Assignment 3/SourabhA3_1.html]: this `</ol>` is required

**Pull request merged**